easybuild/easyconfigs/h/hipify-clang/hipify-clang-4.2.0-gcccuda-2020b.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'CMakeMake'
+
+name = 'hipify-clang'
+version = '4.2.0'
+
+homepage = 'https://github.com/ROCm-Developer-Tools/HIPIFY'
+description = """Hipify-clang is a clang-based tool for translating CUDA
+  sources into HIP sources. It translates CUDA source into an abstract syntax
+  tree, which is traversed by transformation matchers. After applying all the
+  matchers, the output HIP source is produced."""
+
+toolchain = {'name': 'gcccuda', 'version': '2020b'}
+
+source_urls = ['https://github.com/ROCm-Developer-Tools/HIPIFY/archive/refs/tags/']
+sources = ['rocm-%(version)s.tar.gz']
+checksums = ['afdc82ae00e14e8e742be6cd47d8fb120d18fc52fe96cba8d8ac4c56176a432e']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('zlib', '1.2.11'),
+]
+
+dependencies = [
+    # ('cuDNN', '8.0.4.30', '-CUDA-11.1.1', True),
+    ('Clang', '11.0.1'),
+    # ('Python', '3.8.6'),
+]
+
+configopts = '-DHIPIFY_CLANG_TESTS=0'
+configopts += ' -DCMAKE_INSTALL_PREFIX=%(installdir)s'
+configopts += ' -DCMAKE_PREFIX_PATH=$EBROOTCLANG'
+# NOTE: The following configuration options are needed if building with tests
+# (as activated by 'runtest' below), for these to work the above commented
+# dependencies need to be uncommented. Currently testing is disabled due to a
+# lack of 'llvm-lit'.
+# configopts += ' -DCUDA_TOOLKIT_ROOT_DIR=$EBROOTCUDA'
+# configopts += ' -DCUDA_DNN_ROOT_DIR=$EBROOTCUDNN'
+# configopts += ' -DPYTHON_EXECUTABLE=$EBROOTPYTHON/bin/python'
+
+# runtest = 'test-hipify'
+
+# For some reason the executable is not placed in 'bin'
```

Comment:
That must be a bug, no? Should we report this upstream?

Perhaps @Flamefire has some feedback here on how to fix this properly in the `CMakeLists.txt`...

Reply:
Reported as https://github.com/ROCm-Developer-Tools/HIPIFY/issues/340

To much work already to fix that, sorry. Essentially you would want to replace `HIPIFY_INSTALL_PATH` at https://github.com/ROCm-Developer-Tools/HIPIFY/blob/397b58a8fc38fa9c5c3252ccfa6e9cd1f0f141a5/CMakeLists.txt#L105 by `bin` and at https://github.com/ROCm-Developer-Tools/HIPIFY/blob/397b58a8fc38fa9c5c3252ccfa6e9cd1f0f141a5/CMakeLists.txt#L109 by `.` but doing the properly with their setting-stuff and HIP integration is more work. So I'm fine with this postinstall step